pitch.md

```diff
@@ -26,38 +111,100 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
 by describing which unicorn properties (horn color, tattoo, etc.) are most 
 important to users and what kinds of values those properties might have._ 
 
+Because music is vastly complex, I would like the users to be able to write
+specific characteristics they want in their music. Users might want
+different lengths of music, and so users might want something like:
+
+`Phrygian scale in measures 9-16`
+
+although this kind of specificity can sometimes be cheating. The above example
+seems ok because it doesn't specify the specific notes or textures that are
+going to be involved, and if the person writing this program is not going to
+be the one sight reading, this kind of specificity is desirable and even needed.
+
```

Comment:
Have you thought of maybe having the user define what they want to practice and then you create many lines of music that randomly test one of the user's definitions to help remove the aspect of cheating. For example, if I want to test the d chord and the c chord and the Phyrgian scale you would create 10 different pieces that test one of those aspects (but I wont know which piece tests what so I isnt "cheating" as much).